src/app.ts

```diff
@@ -20,6 +20,11 @@ const mainWindow = new WindowManager()
 
 enableLiveReload()
 
+if(!app.isDefaultProtocolClient(app.getName())
```

Comment:
For sanity, maybe add a `.toLowerCase()` here since `.getName()` returns capitalised "Monux".